src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -33,7 +33,7 @@
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
             + PREFIX_CLIENTSOURCE + "friends "
-            + PREFIX_CLIENTSOURCE + "owesMoney"
+            + PREFIX_CLIENTSOURCE + "owesMoney "
```

Comment:
I think this needs to be changed. oweMoney doesnt sound like a client source description